src/azure-cli/azure/cli/command_modules/keyvault/_help.py

```diff
@@ -144,6 +144,31 @@
 short-summary: Manage vault network ACLs.
 """
 
+helps['keyvault private-endpoint'] = """
+type: group
+short-summary: Manage vault private endpoint connections.
+"""
+
+helps['keyvault private-endpoint delete'] = """
+type: command
+short-summary: Delete the specified private endpoint connection associated with a Key Vault.
+"""
+
+helps['keyvault private-endpoint show'] = """
+type: command
+short-summary: Show details of a private endpoint connection associated with a Key Vault.
+"""
+
+helps['keyvault private-link-resource'] = """
+type: group
+short-summary: Manage vault private link resources.
+"""
+
+helps['keyvault private-link-resource show'] = """
+type: command
+short-summary: Show the private link resources supported for a Key Vault.
```

Comment:
https://github.com/Azure/azure-cli/blob/c0e345c7719cf20898ad4257171557b8e7e4d955/src/azure-cli/azure/cli/command_modules/network/_params.py#L745

The `az network private-endpoint create -h`'s help message doesn't conform to this command name. @MyronFanQiu